packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -787,7 +792,43 @@ export async function startBrowser(
     ...webdriverOptions,
     ...wdioOptions,
   })) as CompassBrowser;
-  await browser.navigateTo('http://localhost:7777/');
+
+  if (ATLAS_DOMAIN) {
+    // Navigate to a 404 page to set cookies
+    await browser.navigateTo(`https://${ATLAS_DOMAIN}/404`);
+
+    const cookiesFile = process.env.COOKIES_FILE;
+    if (!cookiesFile) {
+      throw new Error(
+        'ATLAS_DOMAIN is set but COOKIES_FILE is not. Please set COOKIES_FILE to the path of the cookies file.'
+      );
+    }
+    const cookies = JSON.parse(await fs.readFile(cookiesFile, 'utf8'));
+
+    for (const cookie of cookies) {
+      if (
+        cookie.name.includes('mmsa-') ||
+        cookie.name.includes('mdb-sat') ||
+        cookie.name.includes('mdb-srt')
```

Comment:
These are the cookies that are relevant for authenticating.

https://github.com/10gen/mms/blob/6d27992a6ab9ab31471c8bcdaa4e347aa39f4013/server/src/features/com/xgen/svc/cukes/helpers/Client.java#L122-L130

Reply:
Can we add this link as a comment here so that we have the reference and documentation in the code?